src/components/ec-dropdown-search/ec-dropdown-search.vue

```diff
@@ -123,6 +124,7 @@
                   ...tooltipOptions,
                   ...item.tooltip,
                 }"
+                :tabindex="!hasLinksInItems ? 0: -1"
```

Comment:
```suggestion
                :tabindex="hasLinksInItems ? -1: 0"
```
It is better to operate with positive conditions, our brain understands those much more easier.

Reply:
infact the first condition is a positive statement, isn't it?

Reply:
The condition is only one here `"hasLinksInItems` and it is negative because of `!` that means `negative`. For that who read this condition easier to see conditions without `!`. So if you could avoid it - it is better to do.

Reply:
ah ok understood. thank you